pandas/io/spss.py

```diff
@@ -46,6 +47,8 @@ def read_spss(
           DataFrame.
 
         .. versionadded:: 2.0
+    kwargs : dict, optional
+        Additional keyword arguments passed to :func:`pyreadstat.read_sav`.
```

Comment:
We need to add a `versionadded` marker like above. This will be added in 3.0 I guess, @lithomas1, can you confirm this is the case please?

Reply:
I've added v3.